crypto/bn/bn_conv.c

```diff
@@ -154,6 +154,10 @@ int BN_hex2bn(BIGNUM **bn, const char *a)
             return 0;
     } else {
         ret = *bn;
+        if (BN_get_flags(ret, BN_FLG_STATIC_DATA)) {
+            ERR_raise(ERR_LIB_BN, BN_R_EXPAND_ON_STATIC_BIGNUM_DATA);
```

Comment:
I'd use ERR_R_PASSED_INVALID_ARGUMENT instead.

Reply:
This error code will be better, Updated.